### PR TITLE
newdevice: Etherwan managed switches

### PR DESCRIPTION
**Pull request by EnzoZafra**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

### CODE REVIEW

**Comment by laf** [comment]
Do all of the switches Etherwan provide run the same OS/Firmware? If not then I'd say the OS name should be updated to be less generic here.

**Comment by EnzoZafra** [comment]
Didn't find much online however I know they supply one MIB for all their switches. The MIB used to be called Etherwan SW72000 which is not a name for any of their hardware so maybe its what they call their OS? Their MIBs are now called 'ewnSwitchM' (Etherwan Switch Management ?) though

File: includes/definitions/etherwan.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+os: etherwan
+text: 'Etherwan Managed Switch'
+type: network
+icon: etherwan
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+    - { graph: device_processor, text: 'CPU Usage' }
+discovery:
+    - sysObjectId:
+        - .1.3.6.1.4.1.2736.1.1
```
Comment (laf) [inline includes/definitions/etherwan.yaml]: We have nothing in stats for this OID, I'd be inclined to change it to `.1.3.6.1.4.1.2736.`

File: includes/polling/os/etherwan.inc.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @package    LibreNMS
+ * @link       http://librenms.org
+ * @copyright  2017 Lorenzo Zafra
+ * @author     Lorenzo Zafra<zafra@ualberta.ca>
+ */
+
+// SNMPv2-MIB::sysDescr.0 = STRING: Etherwan 6TX+2G Managed Switch, Firmware rev: 1.94.3 05/11/15 16:31:18
```
Comment (laf) [inline includes/polling/os/etherwan.inc.php]: Can remove that, we have the test data.

File: includes/polling/os/etherwan.inc.php [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+$description = $poll_device['sysDescr'];
```
Comment (laf) [inline includes/polling/os/etherwan.inc.php]: No point in assigning this variable, just use `$poll_device['sysDescr']` directly below

**Pull request merged**